protobuf/lib/src/protobuf/proto3_json.dart

```diff
@@ -400,3 +406,48 @@ void _mergeFromProto3Json(
 
   recursionHelper(json, fieldSet);
 }
+
+bool _isAsciiLetter(int char) {
+  const lowerA = 97;
+  const lowerZ = 122;
+  const capitalA = 65;
+  const capitalZ = 90;
+
+  return (capitalA <= char && char <= capitalZ) ||
+      (lowerA <= char && char <= lowerZ);
```

Comment:
Or:
```dart  
  char |= lowerA ^ capitalA;
  return lowerA <= char && char <= lowerZ;
```

Reply:
I knew you where the right one to review this!